specification/mzlb_json.schema.json

```
@@ -0,0 +1,141 @@
+        "cvparam": {
+            "type": "object",
+            "description": "A controlled vocabulary term with an optional associated value, which may itself be a cvparam",
+            "required": ["accession", "name"],
+            "properties": {
+                "accession": {
+                    "type": "string",
+                    "description": "The accession number of the external controlled vocabulary term"
+                },
+                "name": {
+                    "type": "string",
+                    "description": "The human readable name of the controlled vocabular term"
+                },
+                "value": {
+                    "oneOf": [
+                        {"type": ["string", "number", "boolean", "null"]}
+                    ]
+                },
+                "value_accession": {
+                    "type": "string",
+                    "description": "The accession number of the external controlled vocabulary term describing the value of this cvparam"
+                },
+                "cv_param_group": {
+                    "type": "string",
+                    "description": "An identifier connecting a group of related parameters"
+                }
+            }
+        },
+        "analyte": {
+            "type": "object",
+            "description": "A molecule that is believed to have generated or contributed to the spectrum",
+            "required": ["attributes", "id"],
+            "properties": {
+                "attributes": {
+                    "type": "array",
+                    "description": "A list of controlled vocabulary terms describing the analyte",
+                    "items": {
+                        "$ref": "#/definitions/cvparam"
+                    }
+                },
+                "id": {
+                    "type": "string",
+                    "description": "A short identifier unique within the spectrum entry"
+                }
+            }
+        },
+        "spectrum": {
+            "type": "object",
+            "description": "A mass spectrum entry in the library",
+            "required": [
+                "attributes",
+                "analytes",
+                "mzs",
+                "intensities",
+                "interpretations"
+            ],
+            "properties": {
+                "attributes": {
+                    "type": "array",
+                    "description": "A list of controlled vocabulary terms describing the spectrum",
+                    "items": {
+                        "$ref": "#/definitions/cvparam"
+                    }
+                },
+                "analytes": {
+                    "type": "object",
+                    "description": "The analytes described by this spectrum",
+                    "patternProperties": {
+                        ".{1,}": {
+                                "$ref": "#/definitions/cvparam"
+                            }
+                        }
```

Should we limit the key names to integer numbers only?
```suggestion
                    "patternProperties": {
                        "[0-9]{1,}": {
                            "$ref": "#/definitions/cvparam"
                        }
                    },
                    "additionalProperties": false
```